ui/apps/platform/src/Containers/Clusters/Components/ClusterStatus.tsx

```diff
@@ -24,7 +24,8 @@ type ClusterStatusProps = {
 function ClusterStatus({ healthStatus, isList = false }: ClusterStatusProps): ReactElement {
     const { version } = useMetadata();
 
-    const { overallHealthStatus } = healthStatus;
+    const { overallHealthStatus = 'UNAVAILABLE' } = healthStatus ?? {};
```

Comment:
Unless you know something that I do not `UNINITIALIZED` seems more accurate and consistent:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Clusters/clustersTableColumnDescriptors.js#L74-L76

As far as I know, `UNAVAILABLE` is associated with former secured clusters that have been deleted in Kubernetes environment, but not yet in ACS.

Reply:
Good point; for a cluster that has just been created, the first state it would enter would be UNINITIALIZED, until the user runs the connection script or otherwise connects it.

On the other hand, since this component is used in multiple places, and missing health response at runtime could theoretically occur at any stage of the lifecycle, I opted for "unavailable" as the fallback. I'm probably just paranoid.

It reminds me of a story I heard--can't find a reference easily--of when Apple first launched the Macintosh in 1984. Whenever a user created a new folder, it received the name "Empty Folder". You could put as many files and other folders in it as you liked, but you had to change the name manually. This resulted in lots of lazy people with lots of folders full of stuff but still named "Empty Folder".

Apple eventually changed the default name to "Untitled Folder".

Reply:
Resolving after discussing with Mark in-person